backend/fms_core/serializers.py

```diff
@@ -273,12 +274,16 @@ class Meta:
         depth = 1
 
 class ProjectSerializer(serializers.ModelSerializer):
+    samples = SampleSerializer(read_only=True, many=True)
```

Comment:
I think simply having the sample pk would be more appropriate than the whole samples ...

Reply:
Agree. If you need to access the Samples information from the Project page, you can fetch the Samples from the redux store with their IDs with a method such as 'list()' or worse to worse with 'withItem'